plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -174,6 +175,9 @@ open class DefaultPageCreator(
         }
     }
 
+    private fun Collection<DFunction>.sort() =
```

Comment:
Extension might maybe be better on `Iterable` than `Collection`?
Would it make sense to let `DFunction` impelement `Comaprable` or is this order to specific to the current use case?

Reply:
> Extension might maybe be better on Iterable than Collection?

Sure, changed 👍 

> Would it make sense to let DFunction impelement Comaprable or is this order to specific to the current use case?

We can do that, I have two issues with this approach though:
1) It would be kind of inconsistent with other Documentables that don't implement `Comparable`
2) The order here is introduced only for sensible presentation (eg. we prefer methods alphabetically with less arguments) and not for some sort of actual order. When implementing `Comparable` I feel like one can assume that one method is *bigger* or *smaller* than other (as it would be then possible to do `fun1 >= fun2`) and it may lead to some wrong impressions based on different assumptions what *bigger* means. 

Maybe we should rename `sort` to something like `orderLexicographically()` ?

Reply:
Very good arguments presented ☺️ 
I would be fine with calling it sort. Will merge this if you are either ☺️